src/a11y-dialog.ts

```diff
@@ -290,10 +281,6 @@ if (typeof document !== 'undefined') {
   if (document.readyState === 'loading') {
     document.addEventListener('DOMContentLoaded', instantiateDialogs)
   } else {
-    if (window.requestAnimationFrame) {
-      window.requestAnimationFrame(instantiateDialogs)
-    } else {
-      window.setTimeout(instantiateDialogs, 16)
-    }
```

Comment:
Honestly can‘t remember why this was a thing but it seemed like the way to go? 😅

Reply:
Weird. I meant to add a comment about this in my review but I must not have saved it. I think requestAnimationFrame is excess work here because we're not doing anything that affects render or would risk a forced layout.